### PR TITLE
Changes for installation of test environment 

### PR DESCRIPTION
**Pull request by nunomsfranco**
- Updated travis.yml file for automated tests
- deleted docs/tests/docker-compose.yml there is no need for this docker-compose file, tests can be run with command "sudo docker-compose run web nosetests"
- updated main docker-compose file & ./docsbox Dockerfile

### CODE REVIEW

**Review by numsu — APPROVED** [review]
lgtm, but the build didn't seem to run ok :p

**Comment by numsu** [comment]
Also change the readme's build info to point to the correct one

File: .travis.yml [hunk 1]
```diff
@@ -1,29 +1,19 @@
...
-  - docker build --no-cache --rm --tag ${BACKEND_IMAGE}:latest ./docsbox
-  - docker push ${BACKEND_IMAGE}:latest
-  - docker pull ${PROXY_IMAGE}
-  - docker build --no-cache --rm --tag ${PROXY_IMAGE}:latest ./nginx
-  - docker push ${PROXY_IMAGE}:latest
-
+- docker login -u ${DOCKER_USER} -p ${DOCKER_PASS}
+- docker pull ${IMAGE_REPO}
+- docker build --no-cache --rm --tag ${IMAGE_REPO}:latest ./docsbox
+- docker push ${IMAGE_REPO}:latest
```
Comment (numsu) [inline .travis.yml]: Add a specific version to the environment variables to push.

**Review by numsu — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (nunomsfranco) [reply .travis.yml]: I added a new variable so it can changed easily

**Review by numsu — APPROVED** [review]

**Pull request merged**